src/view.js

```diff
@@ -200,13 +205,32 @@ export class View {
   }
 
   #changePlayTonesButtonText(playTonesButton) {
-    if (this.isPlayTonesButtonClicked) {
+    if (this.#isPlayTonesButtonClicked) {
       playTonesButton.textContent = 'Replay tones';
     }
   }
 
+  #playNotesDifferentlyDependingOnGameLeve(level) {
```

Comment:
```suggestion
  #playNotesForGameLevel(level) {
```
Small suggestion to make this a tiny bit more precise, but please feel free to ignore if you feel otherwise 😄

Reply:
It clearly shows I had no good idea about naming this function haha. Thank you for your suggestion 😄